develop/devguide/Automation/Howto/Getting_Started_with_the_IAS_Toolkit.md

```diff
@@ -72,10 +72,9 @@ public class Script
     {
         // DO NOT REMOVE THIS COMMENTED-OUT CODE OR THE SCRIPT WON'T RUN!
```

Comment:
```suggestion
        // DO NOT REMOVE THIS COMMENT OR THE SCRIPT WON'T RUN!
```

Reply:
I only saw this suggestion after merging; I'll add it in a separate commit in a moment.

Reply:
Sorry, I only noticed later that this part could be improved